decred_wallet/view_controller/WalletLogViewController.swift

```diff
@@ -7,13 +7,17 @@
 // license that can be found in the LICENSE file.
 
 import UIKit
+import JGProgressHUD
 
 class WalletLogViewController: UIViewController {
     @IBOutlet weak var logTextView: UITextView!
-    
+    static var progressHud = JGProgressHUD(style: .light)
     override func viewDidLoad() {
         super.viewDidLoad()
         self.navigationItem.title = "Wallet Log"
+        WalletLogViewController.progressHud = Utils.showProgressHud(withText: "Loading log...")
+    }
+    override func viewDidAppear(_ animated: Bool) {
         self.logTextView.text = WalletLogViewController.readLog()
```

Comment:
Why are we reading the log in `viewDidAppear` instead of in `viewDidLoad`?

Reply:
For better transition when moving into the controller.